timber-lint/build.gradle

```diff
@@ -8,7 +8,10 @@ configurations {
 }
 
 dependencies {
-  compile deps.lintapi
+  compile(deps.lintapi) {
+    exclude group: 'com.android.tools.external.com-intellij', module: 'uast'
```

Comment:
com.android.tools.lint:lint-api:25.2.0 depends on com.android.tools.external.com-intellij:uast:145.597.3, but only com.android.tools.external.com-intellij:uast:145.597.4 resolves on Maven Central.  Updating allprojects.repositories from mavenCentral() to jcenter() fails to resolve com.android.tools.lint:lint-tests:25.2.0.

Another option would be to include both mavenCentral() and jcenter() in allprojects.repositories.  Thoughts?